core/src/Dscp/Config/Util.hs

```diff
@@ -119,7 +122,7 @@ fillExpandedConfig filler cfg = flip rreplace cfg <$> filler (rcast cfg)
 -- | CLI parser for config parameters.
 configParamsParser :: Opt.Parser ConfigParams
 configParamsParser = do
-    cpPath <- confPathParser
+    cpPaths <- NonEmpty.some confPathParser
```

Comment:
`--help` should tell that this option can take multiple values and that the order of configs matters. Please update.

Reply:
Done.